hw1/ex2/ex2.py

```diff
@@ -0,0 +1,176 @@
+
+    def connect(self) -> bool:
+        if self.redis_client is None:
+            self.redis_client = redis.Redis(
+                host=self.host,
+                username=self.user,
+                port=self.port,
+                password=self.password
+                )
+            self.connected = self.redis_client.ping()
+            return self.connected
+        return False
+
+    def disconnect(self) -> bool:
+        if self.redis_client is not None:
+            self.redis_client.close()
+            self.redis_client = None
+            self.connected = False
+            return True
+        return False
+
+    def is_connected(self) -> bool:
+        if self.redis_client is not None:
+            self.connected = self.redis_client.ping()
+            return self.connected
+        return False
+
+    def create_ts(self, key: str,
+            compression: bool=True,
+            retention_msecs: int=0) -> bool:
+        if self.is_connected():
+            try:
+                self.redis_client.ts().create(
+                    key=key,
+                    compression=compression,
+                    retention_msecs=retention_msecs
+                    )
+                return True
+            except redis.ResponseError:
+                pass
+        return False
+
+    def delete_ts(self, key: str) -> bool:
+        if self.is_connected():
+            try:
+                self.redis_client.delete(key)
+                return True
+            except redis.ResponseError:
+                pass
+        return False
+
+    def add_value_ts(self, key: str,
+            timestamp: int,
+            value: int) -> bool:
+        if self.is_connected():
+            try:
+                self.redis_client.ts().add(
+                    key=key,
+                    timestamp=timestamp,
+                    value=value
+                    )
+                return True
+            except redis.ResponseError:
+                pass
+        return False
+    
+    def create_rule_ts(self, source_key: str,
+                dest_key: str,
+                bucket_size_msec: int,
+                aggregation_type: str) -> bool:
+        if self.is_connected():
+            try:
+                self.redis_client.ts().createrule(
+                    source_key=source_key,
+                    dest_key=dest_key,
+                    bucket_size_msec=bucket_size_msec,
+                    aggregation_type=aggregation_type   
+                    )
+                return True
+            except redis.ResponseError:
+                pass
+        return False
+
+
+def main(args: dict) -> None:
+    MAC_ADDRESS = hex(uuid.getnode())
+    TS_BATTERY = f'{MAC_ADDRESS}:battery'
+    TS_POWER = f'{MAC_ADDRESS}:power'
+    TS_PLUGGED_SEC = f'{MAC_ADDRESS}:plugged_seconds'
+    db = RedisDB(
+        host=args.host, 
+        port=args.port,
+        user=args.user,
+        password=args.password
+        )
+    print(db.connect())
```

Comment:
Should comment in the output meaning of this printed boolean, or remove this line

Reply:
comment output